This is a testharness.js-based test.
Harness Error. harness_status.status = 2 , harness_status.message = 
[TIMEOUT] navigate() to "javascript:void 0" with { history: 'replace' }
[NOTRUN] navigate() to "javascript:'foo'" with { history: 'replace' }
Harness: the test ran to completion.

